docs/source/api/commands.rst

```diff
@@ -1,47 +1,641 @@
+| Domain ID    | target domain to make relation with    | should be created before the account, `[0-9A-Za-z]{1,9}`                                                                                                                                                                  |
++--------------+----------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Main pubkey  | first public key to add to the account | ed25519 public key                                                                                                                                                                                                        |
++--------------+----------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Validation
+^^^^^^^^^^
+
+1. Transaction creator has permission to create an account
+2. Domain, passed as domain_id, has already been created in the system
+3. Such public key has not been added before as first public key of account or added to a multi-signature account
+
 Create asset
 ------------
 
+Purpose
+^^^^^^^
+
+The purpose of сreate asset command is to create a new type of asset, unique in a domain.
+An asset is a countable representation of a commodity.
+
+Schema
+^^^^^^
+
+.. code-block:: proto
+
+    message CreateAsset {
+        string asset_name = 1;
+        string domain_id = 2;
+        uint32 precision = 3;
+    }
+
+Structure
+^^^^^^^^^
+
++------------+-------------------------------------+--------------------------------------------------------+
+| Field      | Description                         | Constraint                                             |
++============+=====================================+========================================================+
+| Asset name | domain-unique name for asset        | `[A-Za-z0-9]{1,9}`                                     |
++------------+-------------------------------------+--------------------------------------------------------+
+| Domain ID  | target domain to make relation with | should be created before the asset, `[A-Za-z0-9]{1,9}` |
++------------+-------------------------------------+--------------------------------------------------------+
+| Precision  | number of digits after comma/dot    | 0 <= precision <= uint32 max                           |
++------------+-------------------------------------+--------------------------------------------------------+
+
+Validation
+^^^^^^^^^^
+
+1. Transaction creator has permission to create assets
+2. Asset name is unique per domain
```

Comment:
Maybe `within` or `in`.

Reply:
Within means ‘inside or not further than a particular area or space’. Since domain is not a space or area this is not applicable. 
Per is not a good choice also, I will put in.